scheduler/src/cook/rest/api.clj

```diff
@@ -1959,7 +1979,10 @@
           job-asserts (map (fn [j] [:entity/ensure-not-exists [:job/uuid (:uuid j)]]) jobs)
           [commit-latch-id commit-latch] (make-commit-latch)
           db (d/db conn)
-          job-txns (mapcat (partial make-job-txn pool commit-latch-id db) jobs)
+          job-txns (mapcat
+                     (fn [{:keys [job] :as job-pool-name-map}]
+                       (make-job-txn job-pool-name-map commit-latch-id db job))
```

Comment:
This call is weird. You're extracting job from the map in the above line, then passing it as an argument. I think you can just strip the job argument out and have make-job-txn extract it.

Reply:
True, fixed